### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by elliotBraem**
[PREVIEW](https://test.nearbuilders.org/projects)

- [x] _Resolves #179_
- [x] _Resolves #170_
- [x] _Resolves #185_
- [ ] _Resolves #171_

### TODO

- [x] Create Project
- [x] Clicking a project should route to project page (see [buildbox](https://github.com/NEARBuilders/buildbox), use this sample data) @Jikugodwill

### CODE REVIEW

File: apps/builddao/widget/components/modals/CreateProject.jsx [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+    padding: 0;
+    margin: 0;
+  }
+  .err-p_id {
+    margin-top: -20px;
+  }
+`;
+
+function onCreateProject() {
+  const projectID = normalize(title);
```
Comment (elliotBraem) [inline apps/builddao/widget/components/modals/CreateProject.jsx:227]: Screenshot 2024-04-22 at 9 23 45 AM

When saving a project: 

projectId is undefined, also the key builddao.testnet_project_ is missing

File: apps/builddao/widget/components/modals/CreateProject.jsx [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+    profileImage: avatar,
+    backgroundImage: coverImage,
+    projectAccountId: projectAccount,
+    teamSize,
+  };
+  const data = {
+    project: {
+      [projectID]: {
+        "": JSON.stringify(project),
+        metadata: project,
```
Comment (elliotBraem) [inline apps/builddao/widget/components/modals/CreateProject.jsx:248]: Lets have metadata reflect the metadata standard: https://github.com/NearSocial/standards/blob/main/types/common/Metadata.md

AKA, should be an object with name, description, tags, linktree (gihub, telegram, website, etc), image, and backgroundImage

Anything that is not this metadata should not be saved in metadata

File: apps/builddao/widget/components/modals/CreateProject.jsx [hunk 3]
```diff
@@ -0,0 +1,486 @@
...
+      },
+    },
+    [app]: {
+      project: {
+        [`${context.accountId}_project_${projectID}`]: "",
+      },
+    },
+  };
+  if (projectAccount.includes(".sputnik-dao.near")) {
```
Comment (elliotBraem) [inline apps/builddao/widget/components/modals/CreateProject.jsx:257]: Can we make this create modal be more half n half? Move more items to the right
Screenshot 2024-04-22 at 9 29 16 AM

File: apps/builddao/widget/components/modals/CreateProject.jsx (hunk 1 above)
Comment (Megha-Dev-19) [reply apps/builddao/widget/components/modals/CreateProject.jsx]: I think this is done too, looking at your last commit

File: apps/builddao/widget/components/modals/CreateProject.jsx (hunk 3 above)
Comment (Megha-Dev-19) [reply apps/builddao/widget/components/modals/CreateProject.jsx:272]: I have sent to this Emmanuel, will update after the design is updated.

**Pull request merged**